src/gasman.c

```diff
@@ -1796,14 +1800,26 @@ static void GenStackFuncBags(void)
     top = (Bag*)((void*)&top);
     if ( StackBottomBags < top ) {
         for ( i = 0; i < sizeof(Bag*); i += StackAlignBags ) {
-            for ( p = (Bag*)((char*)StackBottomBags + i); p < top; p++ )
-                MarkBag( *p );
+            for ( p = (Bag*)((char*)StackBottomBags + i); p < top; p++ ) {
```

Comment:
```suggestion
            for (p = (Bag *)((char *)StackBottomBags + i); p < top; p++) {
```
Maybe `git clang-format -i` those commits?